.github/workflows/ecl2df.yml

```diff
@@ -24,6 +24,7 @@ jobs:
     strategy:
       matrix:
         python-version: ['3.6', '3.7', '3.8', '3.9']
+        install-ert: [true, false]
```

Comment:
Do we need to test this for every python-version? (thinking of electricity consumption..)

Reply:
We can limit it, not sure what would be the preferred GitHub workflow syntax?

I see https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-additional-values-into-combinations is a concept - not entirely clear to me what happens when doing `if matrix.some_variable` in branches where the new variable is not added :thinking: 

Alternatively wright the combinations fully out.